kyo-data/shared/src/main/scala/kyo/Render.scala

```diff
@@ -16,63 +16,56 @@ sealed trait LowPriorityRenders:
 object Render extends LowPriorityRenders:
     inline def apply[A](using r: Render[A]): Render[A] = r
 
+    def instance[A](impl: A => Text): Render[A] =
+        new Render[A]:
+            def asText(value: A): Text = impl(value)
+
     def asText[A](value: A)(using r: Render[A]): Text = r.asText(value)
 
     import scala.compiletime.*
 
-    @nowarn("msg=anonymous")
     private inline def sumRender[A, M <: scala.deriving.Mirror.ProductOf[A]](label: String, mir: M): Render[A] =
         val shows = summonAll[Tuple.Map[mir.MirroredElemTypes, Render]]
-        new Render[A]:
-            def asText(value: A): String =
-                val builder = java.lang.StringBuilder()
-                builder.append(label)
-                builder.append("(")
-                val valIter                         = value.asInstanceOf[Product].productIterator
-                val showIter: Iterator[Render[Any]] = shows.productIterator.asInstanceOf
-                if valIter.hasNext then
-                    builder.append(showIter.next().asText(valIter.next()))
-                    ()
-                while valIter.hasNext do
-                    builder.append(",")
-                    builder.append(showIter.next().asText(valIter.next()))
-                    ()
-                end while
-                builder.append(")")
-                builder.toString()
-            end asText
-        end new
+        Render.instance: (value: A) =>
```

Comment:
Do we need `Render.instance`? I think the compiler can materialize `Render` directly from an anonymous function

Reply:
When I try to to make `Render` in inline methods via anonymous function I get compiler warn:

```
An inline given alias with a function value as right-hand side can significantly increase
generated code size. You should either drop the `inline` or rewrite the given with an
explicit `apply` method.
```

May be we can just ignore this warn, not sure.